hudi-utilities/pom.xml

```diff
@@ -443,6 +443,23 @@
       <artifactId>aws-java-sdk-sqs</artifactId>
       <version>${aws.sdk.version}</version>
     </dependency>
+
+    <!-- start: GCS Incremental Ingestion -->
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-pubsub</artifactId>
+      <version>1.120.0</version>
```

Comment:
we can declare the version at the top just  how we do for other dependencies. take a look at `aws.sdk.version`

Reply:
Done.